home-assistant-addon/config.json

```diff
@@ -48,6 +49,7 @@
   },
   "schema": {
     "log_level": "list(trace|debug|info|notice|warning|error|fatal)?",
+    "jwt_secret": "password?",
```

Comment:
JWT should not be optional. This also implies that we can now assume it is always set in our scripts etc.